ext/standard/uniqid.c

```diff
@@ -72,12 +74,17 @@ PHP_FUNCTION(uniqid)
 	gettimeofday((struct timeval *) &tv, (struct timezone *) NULL);
 	sec = (int) tv.tv_sec;
 	usec = (int) (tv.tv_usec % 0x100000);
+	php_random_bytes(bytes, UNIQID_RAND_CHARS*2, 1);
+	for (i = 0; i < UNIQID_RAND_CHARS; i++) {
```

Comment:
I'm by no means knowledgeable on security (in fact I know almost nothing about it). But doesn't iterating 't building up the uniqid like this (assuming that's what it's doing) open up a user to timing attacks?